File: back/src/main/java/mitei/mitei/political/balancesheet/manage/kanrensha/dto/wktbl_min/UpdateWkTblMinPoliOrgResultDto.java
package mitei.mitei.political.balancesheet.manage.kanrensha.dto.wktbl_min;

import java.io.Serializable;

import mitei.mitei.political.balancesheet.manage.kanrensha.dto.FrameworkMessageAndResultDto;
import mitei.mitei.political.balancesheet.manage.kanrensha.entity.WkTblPartnerPoliOrgAddMinEntity;

/**
 * 政治団体一括登録マスタ最小ワークテーブル更新CapsuleDto
 */
public class UpdateWkTblMinPoliOrgResultDto extends FrameworkMessageAndResultDto // NOPMD DataClass
        implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 編集対象Entity */
    private WkTblPartnerPoliOrgAddMinEntity wkTblPartnerPoliOrgAddMinEntity = new WkTblPartnerPoliOrgAddMinEntity();

    /**
     * 編集対象Entityを取得する
     *
     * @return 編集対象Entity
     */
    public WkTblPartnerPoliOrgAddMinEntity getWkTblPartnerPoliOrgAddMinEntity() {
        return wkTblPartnerPoliOrgAddMinEntity;
    }

    /**
     * 編集対象Entityを設定する
     *
     * @param wkTblPartnerPoliOrgAddMinEntity 編集対象Entity
     */
    public void setWkTblPartnerPoliOrgAddMinEntity(
            final WkTblPartnerPoliOrgAddMinEntity wkTblPartnerPoliOrgAddMinEntity) {
        this.wkTblPartnerPoliOrgAddMinEntity = wkTblPartnerPoliOrgAddMinEntity;
    }

}

File: front/src/dto/wktbl_min/updateWkTblMinPoliOrgResultDto.ts
import type WkTblPartnerPoliOrgAddMinInterface from "../../entity/wkTblPartnerPoliOrgAddMin";
import WkTblPartnerPoliOrgAddMinEntity from "../../entity/wkTblPartnerPoliOrgAddMin";
import FrameworkMessageAndResultDto from "../frameworkMessageAndResultDto";

export default interface UpdateWkTblMinPoliOrgResultInterface {

}
/**
 * 政治団体一括登録マスタ最小ワークテーブル更新ResultDto
 */
export default class UpdateWkTblMinPoliOrgResultDto extends FrameworkMessageAndResultDto
    implements UpdateWkTblMinPoliOrgResultInterface {

    /** 編集対象Entity */
    wkTblPartnerPoliOrgAddMinEntity: WkTblPartnerPoliOrgAddMinInterface;

    constructor() {
        super();
        this.wkTblPartnerPoliOrgAddMinEntity = new WkTblPartnerPoliOrgAddMinEntity();
    }
}
